tests/NewCypress/cypress/pages/trusts/trustDetailsPage.ts

```diff
@@ -0,0 +1,50 @@
+class TrustDetailsPage {
+
+    elements = {
+        detailsHeader: () => cy.get('header > .govuk-heading-l'),
+        trustDetailSummaryCard: () => cy.get(':nth-child(2) > .govuk-summary-card'),
+        detailsSummaryCardContentBox: () => cy.get(':nth-child(2) > .govuk-summary-card > .govuk-summary-card__content'),
+        referenceNumberSummaryCard: () => cy.get(':nth-child(3) > .govuk-summary-card'),
+        referenceNumberSummaryCardContentBox: () => cy.get(':nth-child(3) > .govuk-summary-card > .govuk-summary-card__content'),
+
```

Comment:
Many of these selectors are brittle, when content moves on the page they will break and it won't necessarily be obvious